Maestro/subscriptions.json

```diff
@@ -83,6 +89,16 @@
           "maestroDelay": "00:05:00"
         }
       ]
+    },
+    {
+      "path": "https://github.com/dotnet/versions/blob/master/build-info/dotnet/cli/rel/1.0.0/Latest_Packages.txt",
```

Comment:
Agreed, this needs the change you mentioned.  I just wanted to get the ball rolling on this :smile: The current way the docker builds are getting triggered are lagging behind the tip builds as well because they are getting triggered off the Ubuntu builds rather than in FinalizeBuild.  I will update when https://github.com/dotnet/cli/pull/3270 is completed.